extensions/pkg/controller/worker/templates/machineclasses.tpl.yaml

```
@@ -14,6 +14,8 @@ spec:
     listKind: MachineClassList
     plural: machineclasses
     singular: machineclass
+    shortNames:
+      - machclass
```

Cosmetic suggestion to prevent no-op diff when comparing with https://github.com/gardener/machine-controller-manager/blob/0d00e1426dfee6b47a5a952b91402db867d16436/kubernetes/crds/machine.sapcloud.io_machineclasses.yaml.

```suggestion
    shortNames:
    - machclass
    singular: machineclass
```